domain-cc/cc-app/src/python_src/pydantic_models.py

```diff
@@ -1,9 +1,11 @@
 from typing import Optional
 
+from fastapi import HTTPException
 from pydantic import BaseModel, root_validator
+from pydantic.types import conlist
 
 
-class Claim(BaseModel):
+class FlattenedSingleIssueClaim(BaseModel):
     claim_id: int
```

Comment:
Does this refer to the VA.gov claim ID or the VBMS claim ID? If the former, how will the VBMS claim ID be obtained?

Reply:
this contains the vagov_claim_id, I will rename accordingly